indexer/bin/indexer.js

```diff
@@ -40,6 +41,9 @@ export default {
       return new Response('Unauthorized', { status: 401 })
     }
 
+    assert(env.DEFAULT_LOCKUP_PERIOD_DAYS)
+    assert(env.FILECOIN_GENESIS_BLOCK_TIMESTAMP_MS)
```

Comment:
aren't those values part of the `env` type and therefore don't need to be asserted?